### PR TITLE
Support for ExpressRoute direct MACsec SCI state

### PR DESCRIPTION
**Pull request by jake-scott**
Fixes 23624

### CODE REVIEW

File: website/docs/r/express_route_port.html.markdown [hunk 1]
```diff
@@ -76,6 +76,8 @@ A `link` block supports the following:
 
 * `macsec_cak_keyvault_secret_id` - (Optional) The ID of the Key Vault Secret that contains the Mac security CAK key for this Express Route Port Link.
 
+* `macsec_sci_enabled` - (Optional) Enable Secure Channel Identifier on the Express Route Port Link.
```
Comment (catriona-m) [inline website/docs/r/express_route_port.html.markdown:79]:
```suggestion
* `macsec_sci_enabled` - (Optional) Should Secure Channel Identifier on the Express Route Port Link be enabled? Defaults to `false`.
```

**Review by catriona-m — CHANGES_REQUESTED** [review]
Hi @jake-scott - I've just left one minor comment on the docs but otherwise this is looking good! Thanks!

File: website/docs/r/express_route_port.html.markdown (hunk 1 above)
Comment (jake-scott) [reply website/docs/r/express_route_port.html.markdown]: Thanks - update made and pushed.

**Review by catriona-m — APPROVED** [review]
Thanks @jake-scott LGTM!

**Pull request merged**